src/main/java/qna/domain/BaseEntity.java

```diff
@@ -0,0 +1,51 @@
+package qna.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+public class BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @CreatedDate
+    private LocalDateTime createAt;
+
+    @LastModifiedDate
+    private LocalDateTime updateAt;
+
+    private Long createId;
+
+    private Long updateId;
+
+    public LocalDateTime getCreatedAt() {
+        return createAt;
+    }
+
+    public LocalDateTime getUpdatedAt() {
+        return updateAt;
+    }
+
+    public Long getCreatedId() {
+        return createId;
+    }
+
+    public Long getUpdatedId() {
+        return updateId;
+    }
+
+    public void setCreatedAt(LocalDateTime createAt) {
+        this.createAt = createAt;
+    }
+
+    public void setUpdatedAt(LocalDateTime updateAt) {
+        this.updateAt = updateAt;
+    }
+}
```

Comment:
```suggestion
@EntityListeners(AuditingEntityListener.class)
@MappedSuperclass
public abstract class BaseEntity {

    @CreatedDate
    @Column(nullable = false)
    private LocalDateTime createdAt;

    @LastModifiedDate
    private LocalDateTime updatedAt;

    public LocalDateTime getCreatedAt() {
        return createdAt;
    }

    public LocalDateTime getUpdatedAt() {
        return updatedAt;
    }
}
```
음 BaseEntity에 식별자를 두시는거보다는 Entity 마다 식별자 전략이 다를수도 있어서 개별적으로 추가해주는걸 추천드려요

미션의 요구사항의 DDL을 보면 createId, updateId는 없어도 될것같습니다
Entity Auditing을 위해서  `@EntityListeners(AuditingEntityListener.class)` 추가해주세요